bitcoinspv/clients/sui/sui_spv_client.go

```diff
@@ -48,16 +53,45 @@ func NewSPVClient(
 	if signer == nil {
 		return nil, ErrSignerNill
 	}
-	if lightClientObjectID == "" || lightClientPackageID == "" {
-		return nil, ErrEmptyObjectID
+
+	lcPackageID, err := sui.PackageIdFromHex(lightClientPackageIDHex)
+	if err != nil {
+		return nil, err
+	}
+
+	lcObjectID, err := sui.ObjectIdFromHex(lightClientObjectIDHex)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: handle this in other PR
```

Comment:
@sczembor and I discuss and think we should create context in the `cli` package and leave it for other tasks.
cc: @robert-zaremba